app/build.gradle.kts

```
@@ -70,11 +70,11 @@ dependencies {
     implementation("androidx.startup:startup-runtime:1.2.0")
 
     // Android architecture components
-    implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.9.2")
+    implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.9.3")
     implementation("androidx.lifecycle:lifecycle-extensions:2.2.0")
-    annotationProcessor("androidx.lifecycle:lifecycle-compiler:2.9.2")
+    annotationProcessor("androidx.lifecycle:lifecycle-compiler:2.9.3")
 
     // Third-party libraries
     implementation("me.zhanghai.android.materialratingbar:library:1.4.0")
-    implementation("com.github.bumptech.glide:glide:4.16.0")
+    implementation("com.github.bumptech.glide:glide:5.0.0")
```

I think Glide published `5.0.0` by mistake? They were working on 5.0.0-rc2 in https://github.com/bumptech/glide/pull/5574, but 5.0.0 is live on mavenCentral: https://central.sonatype.com/artifact/com.github.bumptech.glide/glide/5.0.0/overview

```suggestion
    implementation("com.github.bumptech.glide:glide:4.16.0")
```